docs/symfony/get-it-started.md

```diff
@@ -84,7 +84,36 @@ class Payment extends BasePayment
 }
 ```
 
-next, you have to add mapping information, and configure payum's storages:
+If you use YAML mapping in your bundle:
+
+```yml
+# src/Acme/PaymentBundle/Resources/config/doctrine/Payment.orm.yml
+
+Acme\PaymentBundle\Entity\Payment:
+    type: entity
+    table: payment
+
+    id:
+        id:
+            type: integer
+            generator:
+                strategy: IDENTITY
```

Comment:
there are annotations on the model that do the same thing.

Reply:
Yeah, sure. But if you have any YAML files in your bundle, annotations won't work anymore, see https://stackoverflow.com/a/26141187/1668200

That's why I suggested to show YML, XML, and Annotations in parallel.